web/src/components/storage/VolumeForm.jsx

```diff
@@ -719,17 +719,26 @@ export default function VolumeForm({ id, volume: currentVolume, templates = [],
 
   const { fsType, snapshots } = state.formData;
 
+  const ShowMountPointSelector = () => (
+    <MountPointFormSelect
+        id="mountPoint"
+        value={state.formData.mountPoint}
+        onChange={changeVolume}
+        volumes={currentVolume ? [currentVolume] : templates}
+        isDisabled={currentVolume !== undefined}
+    />
+  );
+
+  const ShowMountPoint = () => <p>{state.formData.mountPoint}</p>;
+
   return (
     <Form id={id} onSubmit={submitForm}>
       <FormGroup isRequired label={_("Mount point")} fieldId="mountPoint">
-        {currentVolume !== undefined} && <p> {state.formData.mountPoint} </p>
-        {/* <MountPointFormSelect
-          id="mountPoint"
-          value={state.formData.mountPoint}
-          onChange={changeVolume}
-          volumes={currentVolume ? [currentVolume] : templates}
-          isDisabled={currentVolume !== undefined}
-        /> */}
+        <If
```

Comment:
I suspect linter (ESLint in this case) is going to complain here, because indentation. You can check by yourself by running `npm run eslint`.